### ISSUE TITLE
[Workshop] Introduction to Accessibility

### ISSUE THREAD

**Issue by ChrisOwen101** [CodeYourFuture/syllabus#5]
## Aims & Learning Objectives

- List the 4 main groups of disabilities (visual, hearing, motor, cognitive)
- Define what accessibility is
- Recall some specific conditions for each categories
- Define what an invisible disability  is and give some examples
- Define situations where you can experience an accessibility issue, for touch, see, hear and speak or a combination of more than one of them (could be done as homework)
- List some reasons why accessibility is important to consider
- Illustrate what each letter in SCULPT mean
- Apply advice to check colours and give good text links and alt texts
- Create an alternative text when sharing photo on social media.
- Identify good and bad practice in social media
- List the 4 principles of WCAG and define them in their own words
- Recall the 3 conformance levels and which one to aim for
- Identify situations where respecting the guidelines is not enough for a good user experience

## Prior Knowledge

Students will have knowledge of HTML & CSS

## Completion

When completed the lesson plan, slides and exercises should be added to the Workshop section of the Syllabus

**Comment by ChrisOwen101** [comment]
@all-contributors please add @stephanie-K for Workshop

**Comment by ChrisOwen101** [comment]
All Contributors please add Stéphanie Krus for Tutorial

**Comment by ChrisOwen101** [comment]
@all-contributors please add @stephanie-K  for Tutorial

**Comment by ChrisOwen101** [comment]
@all-contributors please add @ChrisOwen101 for docs

**Comment by ChrisOwen101** [comment]
@all-contributors add @stephanie-K as Tutorials

**Comment by ChrisOwen101** [comment]
@all-contributors add @stephanie-K for Tutorials

**Comment by ChrisOwen101** [comment]
@all-contributors add @stephanie-K for Tutorials

**Comment by ChrisOwen101** [comment]
@all-contributors add @stephanie-K for Tutorials

**Comment by ChrisOwen101** [comment]
@all-contributors add @stephanie-K for Tutorials

**Comment by ChrisOwen101** [comment]
@all-contributors add @stephanie-K for a11y
